### ISSUE TITLE
A11y_AzureToolsforVSCode_AzureFunction_DefaultScreen_ScreenReader: The NVDA is not announcing the control type for the "Microsoft" and "Ratings" controls. 

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/vscode-azurefunctions#2948]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#BM_AzureToolsforVSCode_Win32_Aug2021;#Azure Tools for VS
Code;#A11ySev2;#WCAG4.1.2;#Benchmark;#DesktopApp;#Win10;#NVDA;#JAWS;

### Environment Details:
Visual Studio Code
Azure Function
Operating System: Windows 10 Pro 21H1

### Steps to Reproduce:
1. Launch the Visual studio code.
2. TAB to on "Extension" button and search for "Azure Functions" and select it using ENTER key.
3. "Azure Function" screen should open.
4. Start NVDA and TAB to "Microsoft" control.
5. Observe the announcement for "Microsoft" control.

### Actual:
The NVDA is only announcing "Clickable Microsoft" but it does not announce the control type for this "Microsoft" control.

Same issue is also observed with "Ratings" control. 
**This issue is also observed with JAWS.**

### Expected:
The NVDA should announce the appropriate control type for "Microsoft" and "Ratings" controls. 

### User Impact:
Screen reader users won't be able to know control type of  "Microsoft" and "Ratings" controls.

**Comment by ejizba** [comment]
An extension cannot control the marketplace page. This issue should be filed on VS Code: https://github.com/microsoft/vscode/issues

**Comment by 12-shweta** [comment]
@ejizba 
Logged an external bug on the above mentioned URL https://github.com/microsoft/vscode/issues/132645

**Comment by nturinski** [comment]
This was fixed in VS Code via https://github.com/microsoft/vscode/commit/e7482ce3bdbf5719bf169800028b89083880956b.  Can we close this?

**Comment by 12-shweta** [comment]
Verified the bug in the VS code Insiders.
Issue is fixed and now the NVDA announces control type for the controls.
But for the "Microsoft" control it also announces "Microsoft button publisher name"
![Issue](https://user-images.githubusercontent.com/54656883/135384768-aa7a1a0d-c345-49a4-9df2-b72c7b7b5ed5.gif)

**Comment by nturinski** [comment]
Hmm, well this would still be considered an external bug so please log it against VS Code.

**Comment by 12-shweta** [comment]
@nturinski 
Logged an external bug for the "Publisher name" issue:
 https://github.com/microsoft/vscode/issues/134262

**Comment by 12-shweta** [comment]
Verified the external bug [microsoft/vscode#134262](microsoft/vscode#134262) and as per the mentioned comment in external bug, title attribute has been provided due to which it is reading the "Publisher name" along with "Microsoft"
Hence closing the bug.

**Comment by Rama8345** [comment]
**Github Tags:**#Closed;
